### ISSUE TITLE
Installer: untick "take me to the docs" by default

### ISSUE THREAD

**Issue by joozek78** [Starcounter/Starcounter#4005]
1. I have never left it on - I already know where the docs lie and how to access them
1. I have never used a similar feature in other tools / frameworks / platforms I've installed - If I install it, I have probably seen the docs already
1. It's simply annoying to untick it every single time I install SC

**Comment by andwah** [comment]
FYI: it's only when you have a fresh installation, during upgrades this checkbox is not available.

**Comment by joozek78** [comment]
Yeah, but upgrades sometimes fail - especially when the data can't be
migrated (which happen quite often for me). I still had literally never
thought about using this checkbox.

**Comment by andwah** [comment]
Instead of a checkbox we might replace it with link to the docs.

That would make it more user friendly.

**Comment by joozek78** [comment]
+1
